docs/what-is-regression-games.mdx

```diff
@@ -6,16 +6,23 @@ sidebar_label: 'What is Regression Games?'
 
 Regression Games makes it easy to create automated tests for your game.
 
+:::info
+
+These docs cover a few of our core features that you can try now with our open source SDK. For a full demo of all capabilities, please
```

Comment:
Framing these docs as "this is the light version of what you can do, chat with us more to see our other features."

My goal for any recent publicity is not direct usage of the SDK, but rather "book a demo with us", and so you'll see a lot more focus on that in the site. That will change once these docs get all setup.